src/components/layermanager/partials/layermanager.html

```diff
@@ -23,11 +23,12 @@
                 ng-options="(i | gaTimeLabel:layer) for i in layer.timestamps"></select>
       </div>
       <button class="ga-icon ga-btn icon-user"
-              ng-click="showWarning(layer)"></button>
+                ng-click="showWarning(layer)"></button>
       <!--button ng-if="isBodLayer(layer)" class="ga-icon ga-btn icon-info-sign"
-              ng-click="displayLayerMetadata($event, layer)"></button-->
-    </div>
-    <div class="ga-layer-tools">
+                ng-click="displayLayerMetadata($event, layer)"></button-->
+      <button class="ga-icon icon-gear" ng-click="collapsed=!collapsed" href="#subSelection"></button>
```

Comment:
you don't need the `onclick` I think, and the `href`attribute doesn't exist for `<button>`.